lib/mocktopus.rb

```diff
@@ -1,5 +1,9 @@
 require 'logger'
-LOGGER = Logger.new('mocktopus.log', 'daily')
+
+$logger = Logger.new(STDOUT) unless $logger
+
+class ::Logger; alias_method :write, :info; end
```

Comment:
are these aliases still needed since logger is sending everything to stdout now?  I'm assuming this is part of the mentioned monkeyhacks!